coordinator/src/message.rs

```diff
@@ -71,7 +70,13 @@ pub fn spawn_delivering_messages_to_authenticated_users(
     let (fut, remote_handle) = {
         async move {
             while let Some(notification) = receiver.recv().await {
-                let mut conn = pool.get()?;
+                let mut conn = match pool.get() {
+                    Ok(conn) => conn,
+                    Err(e) => {
+                        tracing::error!("Failed to get DB connection: {e:#}");
+                        continue;
+                    },
+                };
```

Comment:
Maybe it would be cleaner here to wrap the inside of the `while` with a `try` block (idk if stable, we could use an async block/closure instead possibly)? that way, `?` would only bubble up to that point.

Reply:
`try` would be super cool, but it's not stable.

I wasn't gonna bother originally, but since you mention it I can just extract the inside of the `while` loop into a function, like in all the other places where I fixed this.

Reply:
See https://github.com/get10101/10101/compare/7b236da36f5196ad73e6dc0da9088c677556d3e1..e0a341c90840f14dbf17b9ae3b4e0b21c59b3aa2.

Reply:
Looks good, that works too!